inscriptions/ISic000548.xml

```diff
@@ -202,59 +207,36 @@
-                    </expan>
-                    Iulia<lb n="3" break="no"/>no
-                    <choice>
-                        <sic>
-                            <expan>
-                                <abbr>evok</abbr>
-                                <ex>ato</ex>
-                            </expan>
-                        </sic>
-                        <corr>evocato</corr>
-                    </choice>
+                    <lb n="1"/><expan><abbr>D</abbr><ex>is</ex></expan> <expan><abbr>m</abbr><ex>anibus</ex></expan>
+                    <lb n="2"/><name><expan><abbr>Tac</abbr><ex>itio</ex></expan></name> <g ref="#interpunct">·</g> <name>Iulia
+                    <lb n="3" break="no"/>no</name> <g ref="#interpunct">·</g> <choice><sic><expan><abbr>evok</abbr><ex>ato</ex></expan></sic><corr>evocato</corr></choice>
                     <lb n="4"/>quondam
-                    <lb n="5"/>
-                    <expan>
-                        <abbr>Aur</abbr>
-                        <ex>elia</ex>
-                    </expan>
-                    Lucil<lb n="6" break="no"/>la coniu<lb n="7" break="no"/>gi karissi
-                    <lb n="8" break="no"/>
-                    <choice>
-                        <sic>mo</sic>
-                        <corr>carissimo</corr>
-                    </choice>
-                    et Iuli<lb n="9" break="no"/>us
-                    Tacitia<lb break="no" n="10"/>nus filius<lb n="11"/>heredes<lb n="12"/>posuerunt<lb n="13"/>merenti.
+                    <lb n="5"/><name><expan><abbr>Aur</abbr><ex>elia</ex></expan></name> <g ref="#interpunct">·</g> <name>Lucil
+                    <lb n="6" break="no"/>la</name> <g ref="#interpunct">·</g> coniu
+                    <lb n="7" break="no"/>gi <g ref="#interpunct">·</g> <choice><sic>karissi
+                    <lb n="8" break="no"/>mo</sic><corr>carissimo</corr></choice> <g ref="#interpunct">·</g> et <g ref="#interpunct">·</g> <name>Iuli
+                    <lb n="9" break="no"/>us</name> <g ref="#interpunct">·</g> <name>Tacitia
+                    <lb break="no" n="10"/>nus</name> <g ref="#interpunct">·</g> filius
+                    <lb n="11"/>heredes
+                    <lb n="12"/>posuerunt
+                    <lb n="13"/>merenti
                 </ab>
-                <!--This EDR text 138306 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
-            </div>
+                           </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app><note>Text of CIL</note></app>
+                </listApp>
+            </div>
+            <div type="translation" xml:lang="en" resp="#GC">
+                <p>To the spirits of the dead. The heirs dedicated (this) for the deserving Tacitus Iulianus Evocatus, his most beloved wife Aurelia Lucilda and their son Iulius Tacitanus.</p>
```

Comment:
if possible, it's good to try to keep tot he approximate order of the Latin, so, I'd suggest:
To the spirits of the dead. To Tacitius Iulianus, a former soldier. Aurelia Lucilla, for her most beloved husband and Iulius Tacitianus, his son, as his heirs set it up for a deserving man.